tesseract/tesseract_planning/tesseract_process_managers/include/tesseract_process_managers/process_generator.h

```diff
@@ -46,7 +46,12 @@ class ProcessGenerator
 public:
   using UPtr = std::unique_ptr<ProcessGenerator>;
 
+  ProcessGenerator() = default;
   virtual ~ProcessGenerator() = default;
+  ProcessGenerator(const ProcessGenerator&) = delete;
+  ProcessGenerator& operator=(const ProcessGenerator&) = delete;
+  ProcessGenerator(ProcessGenerator&&) = delete;
+  ProcessGenerator& operator=(ProcessGenerator&&) = delete;
```

Comment:
Why can't these be copied?

Reply:
It is good to delete the base class copy constructors because copying is slicing. Also I believe adding an atomic implicitly deletes the copy constructor and maybe the move constructor.